app/views/hyrax/homepage/_featured.html.erb

```diff
@@ -1,11 +1,9 @@
 <% presenter = featured.presenter %>
-<li class="featured-item" data-id="<%= presenter.id %>">
-  <div class="main row">
-    <div class="col-sm-3">
-      <%= render_thumbnail_tag presenter, { width: 90 } %>
-    </div>
-    <div class="col-sm-9">
-      <%= render 'hyrax/homepage/featured_fields', featured: presenter %>
-    </div>
-  </div>
-</li>
+<tr class="featured-item" data-id="<%= presenter.id %>">
+  <td class="col-sm-2">
+    <%= render_thumbnail_tag presenter, { width: 90 } %>
+  </td>
+  <td>
```

Comment:
Should this be `col-sm-10`?

Reply:
It could be I suppose, it isn't currently (in the existing code, so I wasn't sure if it was necessary). If you think it's important, I'm happy to get it in there.

Reply:
It appears to be in the existing code to me:
```
<div class="col-sm-9">
```

Reply:
Ah, you're right. I had pulled this over from https://github.com/samvera/hyrax/blob/master/app/views/hyrax/homepage/_recent_document.html.erb where it's missing.

Reply:
It'd be great if we could remove the hard coded width and use CSS